lib/sandthorn_sequel_projection.rb

```diff
@@ -33,17 +32,14 @@ def configure
     end
 
     def start
-      ProcessedEventsTracker.migrate!(configuration.db_connection)
+      ProcessedEventsTracker.migrate!(configuration.db_connection_projections)
     end
 
-    def find_event_store(name)
-      EventStore.new(name)
-    end
   end
 
   class Configuration
 
-    attr_accessor :db_connection, :event_stores, :projections_folder, :batch_size
+    attr_accessor :db_connection_projections, :event_store, :projections_folder, :batch_size
```

Comment:
To me it's the same as naming your database columns like `cars.car_name`, makes the hair on my back stand up :S